upgrading/glusterfs_upgrades.adoc

```diff
@@ -0,0 +1,100 @@
+[[upgrading-openshift-when-using-glusterfs]]
+= Upgrading {product-title} When Using GlusterFS Container Native Storage
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[special-considerations-for-glusterfs]]
+== Special Considerations for GlusterFS
+When upgrading  {product-title}, you must upgrade the set of nodes where
+GlusterFS pods are running.
+
+
+Special consideration must be taken when upgrading these nodes, as `drain` and
+`unschedule` will not terminate and evacuate the GlusterFS pods because they are
+running as part of a daemonset.
+
+There is also the potential for someone to run an upgrade on multiple nodes at
+the same time, which would lead to data availability issues if more than one was
+hosting GlusterFS pods.
+
+Even if a serial upgrade is running, there is no guarantee sufficient time will
+be given for GlusterFS to complete all of its healing operations before
+GlusterFS on the next node is terminated. This could leave the cluster in a bad
+or unknown state. Therefore, the following procedure is recommended.
+
+[[upgrading-openshift-when-running-glusterfs]]
+== Upgrading {product-title} When Using Containerized GlusterFS
+
+. Upgrade the control plane (the master nodes and etcd nodes).
+
+. Upgrade standard infra nodes (router, registry, logging and metrics).
++
+[NOTE]
+====
+If any of the nodes in those groups are running GlusterFS , perform step 4 of
+this procedure at the same time. GlusterFS nodes must be upgraded along with
+other nodes in their class (`app` versus `infra`), one at a time.
+====
+
+. Upgrade standard nodes running application containers.
++
+[NOTE]
+====
+If any of the nodes in those groups are running GlusterFS , perform step 4 of
+this procedure at the same time. GlusterFS nodes must be upgraded along with
+other nodes in their class (`app` versus `infra`), one at a time.
+====
+
+. Upgrade the {product-title} nodes running GlusterFS one at a time.
+
+.. Run `oc get daemonset` to verify the label found under `NODE-SELECTOR` is:
++
```

Comment:
Remove `is` and replace with ``. The default value is `storagenode=glusterfs`.``